### PR TITLE
Update issue template to be more descriptive

### PR DESCRIPTION
**Pull request by willopez**
Impact: minor  
Type: style

Updates issue template to add more details and requirements. The goal set a standard of quality for new issues.

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,16 +1,25 @@
-For bugs, please fill out the three following sections completely. For the ticket to be resolved, you'll want to provide as much information as possible.
+Issue tracker is **ONLY** used for reporting bugs. NO NEW FEATURE REQUESTS ACCEPTED. For new feature navigate to [Request new feature](https://github.com/reactioncommerce/reaction-feature-requests).
 
-Are you looking for help with getting started on Reaction? Please go
-to our [Gitter room](https://gitter.im/reactioncommerce/reaction) instead.
+Are you looking for help with getting started on Reaction? Please visit
+to our [Reaction documentation](https://docs.reactioncommerce.com/reaction-docs/master/getting-started-developing-with-reaction).
```
Comment (zenweasel) [inline .github/issue_template.md]: typo "Please visit to our" should be "Please visit our"

File: .github/issue_template.md [hunk 2]
```diff
@@ -1,16 +1,25 @@
-For bugs, please fill out the three following sections completely. For the ticket to be resolved, you'll want to provide as much information as possible.
+Issue tracker is **ONLY** used for reporting bugs. NO NEW FEATURE REQUESTS ACCEPTED. For new feature navigate to [Request new feature](https://github.com/reactioncommerce/reaction-feature-requests).
```
Comment (zenweasel) [inline .github/issue_template.md]: typo: "For new features", or maybe "to request a new feature"

**Comment by willopez** [comment]
Thanks @zenweasel updated to reflect suggestions.

File: .github/issue_template.md [hunk 3]
```diff
@@ -1,16 +1,24 @@
-For bugs, please fill out the three following sections completely. For the ticket to be resolved, you'll want to provide as much information as possible.
+Issue tracker is **ONLY** used for reporting bugs. NO NEW FEATURE REQUESTS ACCEPTED. For new features navigate to [Request new feature](https://github.com/reactioncommerce/reaction-feature-requests).
```
Comment (machikoyasuda) [inline .github/issue_template.md]: For new features, add a issue in our [Reaction Feature Requests](https://github.com/reactioncommerce/reaction-feature-requests) repository.

File: .github/issue_template.md [hunk 4]
```diff
@@ -1,16 +1,24 @@
...
 
-Are you looking for help with getting started on Reaction? Please go
-to our [Gitter room](https://gitter.im/reactioncommerce/reaction) instead.
+Are you looking for help with getting started on Reaction? Please visit our [Reaction documentation](https://docs.reactioncommerce.com/reaction-docs/master/getting-started-developing-with-reaction).
 
-### Expected behavior
+## Prerequisites
+* [ ] Are you running the latest version?
+* [ ] Are you able to consistently reproduce the issue?
+* [ ] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)
```
Comment (machikoyasuda) [inline .github/issue_template.md]: Love that you added a to do list and added a link to search the issues 👍 👍 👍

**Comment by willopez** [comment]
@machikoyasuda added your suggestion.

**Pull request merged**